pwndbg/commands/dev.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+import argparse
+
+import pwndbg.commands
+import pwndbg.disasm
+import pwndbg.gdblib.nearpc
+
+parser = argparse.ArgumentParser(description="Dump internal PwndbgInstruction attributes.")
+
+# We don't have a parser to pass in true/false in arguments, so there are two args to force the enabling/disabling of emulation
+parser.add_argument(
+    "-e",
+    "--emulate",
+    dest="force_emulate",
+    action="store_true",
+    default=False,
+    help="Force the use of emulation when enhancing the instruction, regardless of global 'emulate' setting.",
+)
+
+parser.add_argument(
+    "-n",
+    "--no-emulate",
+    dest="no_emulate",
+    action="store_true",
+    default=False,
+    help="Disable the use of emulation when enhancing the instruction, regardless of global 'emulate' setting.",
+)
+
+
+@pwndbg.commands.ArgparsedCommand(parser)
```

Comment:
I think it would be good to create a new command category for dev commands, maybe `CommandCategory.DEV` or `CommandCategory.INTERNAL`.

Reply:
Good call